src/components/ApiCall.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/no-array-index-key */
+import React, { useEffect, useState } from 'react';
+import Loading from './Loading';
+
+function ApiCall() {
+  const [quotes, setQuotes] = useState([]);
+  const [error, setError] = useState(false);
+
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+    }, 1500);
+  }, []);
+
+  useEffect(() => {
+    const fetchApi = async () => {
+      const apiKey = 'cGltlxSEokP4E0ug43o13g==0nAuSi7eMBVhj59K';
+      const apiUrl = 'https://api.api-ninjas.com/v1/quotes?category=happiness';
+
+      try {
+        const result = await fetch(apiUrl, {
+          method: 'GET',
+          headers: {
+            'X-Api-Key': apiKey,
+            'Content-Type': 'application/json',
+          },
+        });
+        const data = await result.json();
+        setQuotes(data);
+      } catch (error) {
+        setError(true);
+      }
+    };
+    fetchApi();
+  }, []);
+
+  return (
+    <>
+      {loading ? (
+        <Loading />
+      ) : (
+        <section className="calculator__quotes">
+          {quotes.map((quote, index) => (
+            <div key={index}>
+              <h2 className="calculator__quotes__title">Author: {quote.author}</h2>
+              <p className="calculator__quotes__category">Category: {quote.category}</p>
+              <p className="calculator__quotes__quote">Quote: {quote.quote}</p>
+            </div>
+          ))}
+        </section>
+      )}
+    </>
+  );
+}
+
+export default ApiCall;
```

Comment:
- [ ] Well done, you did a great job but it seems like you made a very small mistake because you a requirement you missed in the project. As per the project requirement, you should add the `ApiCall` component should be called directly inside the `App.js` instead of the `Calculator` component because by doing this if any changes happened on the `ApiCall` component it causes a re-render of the `Calculator` component too. So kindly adhere to the project requirement and implement the things as per the requirements below.

## Requirement

![Screenshot from 2023-08-11 00-08-57](https://github.com/Felipe-Perez-Ferraro/math-magicians/assets/66476329/8e9776a8-31ab-4733-b2c3-97e90f076d76)